protocols/4ce987/4ce987.ot2.py

```diff
@@ -0,0 +1,284 @@
+        pip300.blow_out(istock45.wells(well).top())
+        pip300.drop_tip()
+
+    # initiator in purple to create 0.01x orhton ON
+    pick_up(pip300)
+    pip300.distribute(57, h2o.bottom(26), istock45.wells(cc+ff+eleven),
+                      new_tip='never', disposal_vol=5)
+    pip300.drop_tip()
+
+    for src, dest in zip(aa+dd+gg, cc+eleven+ff):
+        pick_up(pip10)
+        pip10.transfer(3, istock45.wells(src).bottom(8),
+                       istock45.wells(dest), new_tip='never')
+        pip10.drop_tip()
+
+    robot.pause('Please remove plate from slot 5 to mix/vortex mastermixes. \
+    After mixing, replace the plate and click RESUME.')
+
+    # initiator in purple to create 0.01x Crosstalk
+
+    pick_up(pip10)
+    pip10.distribute(3, tr24a.wells('A3'), istock9.wells(cc+ff).top(-4))
+
+    pick_up(pip10)
+    pip10.distribute(3, tr24a.wells('A1'), istock9.wells(ff).top(-4))
+
+    pick_up(pip10)
+    pip10.distribute(3, tr24a.wells('A1'), istock9.wells(eleven[3:]).top(-4))
+
+    pick_up(pip10)
+    pip10.distribute(3, tr24a.wells('A1'), istock9.wells('G11', 'H11').top(-4))
+
+    for well in cc+eleven:
+        pick_up(pip10)
+        pip10.transfer(3, tr24a.wells('A5').bottom(8),
+                       istock9.wells(well).top(-4), new_tip='never')
+        pip10.drop_tip()
+
+    for i, (src1, src2) in enumerate(zip(aa, bb)):
+        d = cc[:i]+cc[i+1:]
+        for well in d:
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src1),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src2),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+
+    for i, (src1, src2) in enumerate(zip(gg, hh)):
+        d = ff[:i]+ff[i+1:]
+        for well in d:
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src1),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src2),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+
+    for i, (src1, src2) in enumerate(zip(dd, ee)):
+        d = eleven[:i]+eleven[i+1:]
+        for well in d:
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src1),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+            pick_up(pip10)
+            pip10.transfer(3, istock9.wells(src2),
+                           istock9.wells(well), new_tip='never')
+            pip10.drop_tip()
+
+    robot.pause('Crosstalk complete. Please remove plate from slot 4 and mix/\
+    vortex. After replacing the plate, click RESUME.')
+
+    # mastermix
+
+    pick_up(pip300)
+    pick_up(pip10)
+
+    mmlist = (tr24b.rows('A')+tr24b.rows('B')+tr24b.rows('C')
```

Comment:
[well for row in tr24b.rows()[:4] for well in row] + tr24a.wells('D3', 'D4', 'D5', 'D6')

Reply:
remove '()' if it's a list